dev/benchmarks/test_apps/stocks/lib/i18n/stock_strings_en.dart

```diff
@@ -2,11 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore: unused_import
```

Comment:
@shihaohong Is there a reason for the unused intl import? (here and below)

Reply:
I believe this is because this file is generated code (by the `gen-l10n` tool). Sometimes, the generated code requires the use of APIs that belong in the `intl` package. It just so happens that in the stocks app, none of the API is used.

Reply:
That being said, we should probably revert the changes to the stocks app? They would eventually be regenerated if gen-l10n was run on the application. We could also look into only conditionally generating the `intl` import only if it's used.

Reply:
makes sense if this file is generated. Maybe we should include a little header in these files to inform people that they are generated :)

Reply:
I'll revert them.